content/pages/qbot.md

```diff
@@ -0,0 +1,55 @@
+
+The name references the tool-maker who provides weapons and gadgets for Agent 007 in the James Bond movies.
+
+## What Q Bot can do
+Q Bot has an evolving set of functions:
+  - Providing to the channel notifications related to your project's Jira tickets - new ticket, new comment, ticket resolved.
+  - Starting a huddle in a channel workspace
+  - Adding people to private channels
+
+It also has a set of 'fun' features, such as rolling dice or flipping a coin.
+
+## Setting up Q Bot in your PMC's channels
+If you want to have Q Bot available in your PMC's channel, create a Jira ticket for Infra with the request. Include this information:
+  - The name of the channel or channels where Q Bot notifications about Jira tickets related to your projects should appear
+  - How you want notifications to appear. There are five options:
+      - No notifications
+      - All notifications
+      - `nodescription`: All notifications without the title of the ticket, but not its description
+      - `nocomments`: No notifications of comments on tickets
+      - `createclose`: Just notifications of ticket creation and resolution
+
+  - Specify who can use the `addme` command -- committers, members of your PMC only, ASF Members only...
+
+To have Q Bot active in a project's private channel, once it is available in your public channels, someone already in the channel has to invite it with the Slack command `/invite @QBot`.
+  
+
+## Talking to Q Bot
+In Slack, there are three ways to talk to Q Bot:
+
+  -  Anywhere in the ASF workspace, type `/qbot` and the command you want to give it. If it is a command which you do not have permission to use where you currently are in Slack, the command will fail.
+  -  In a channel where Q Bot is present, type `qbot` without the slash, and the command.
+  -  In the Slack interface, there is a general menu bar at the far left. The next column is the menu bar for the ASF workspace. At the bottom of the list of your contacts and channels, there is a section for `Apps`. Q Bot appears there as `Qbot`.
+        -  Click that entry to see a display with three tabs.
+        -  Select the `Messages` tab.
+        -  Write your command to Qbot without `/` or 'qbot` at the start.
+        
+
+## Q Bot commands
+Here is what you can do with Q Bot's commands. **Note** we are expanding Q Bot's reach, and this list will expand to match its capabilities.
+
+  - `addme`- provide the Slack ID of the person to add to the private channel. (Without Q Bot, you have to request in a Jira ticket that Infra do this.)
+  - `flip` - Q Bot will flip a coin and tell you whether it came up heads or tails.
+  - `help` - You will see a list of the available commands.
+  - `q` - This command starts a huddle in your current channel, and makes you the administrator. The syntax is `/qbot q start`.
```

Comment:
This one is not so much about starting huddles, but rather about organizing a Q&A queue.

Reply:
Huh. Oh, yes. The thing we have abandoned.

Reply:
Fixed